documentation/developing-your-hyrax-based-app.md

```diff
@@ -269,3 +270,158 @@ proxy_deposit:
 
 If both options exist, whichever option is set from the Administrative Dashboard
 will take precedence.
+
+## Dockerizing a Hyrax Based App for Development
+It is now possible develop a Hyrax based app using a Docker workflow. The process utilizes **docker-compose** to build and run the services necessary for Hyrax based applications.
+
+### Switch to Postgres
+The image that you will build your app's image on uses **Postgres**. If you have just created your Hyrax-based app, for example, you'll need to update your `Gemfile`
+	- `gem "sqlite3", "~> 1.3.0"` ---> `gem "pg"`
+
+### Building the image
+Next you'll want to build the image that your Hyrax container will run from. For that, you will need to create a Dockerfile, build the image from it, and push it to a repository you can fetch from later.
+
+**Build image**
+- Run `cd <hyrax_app_name>`
+- Visit [samveralabs/hyrax](https://hub.docker.com/r/samveralabs/hyrax/tags) and grab the latest image **tag**
+- Run `echo "FROM samveralabs/hyrax:7<tag>" > Dockerfile` to create the Dockerfile
```

Comment:
we are going to need an image tagging strategy, and should update this as soon as we have something figured out. some input: https://cloud.google.com/solutions/best-practices-for-building-containers, https://stevelasker.blog/2018/03/01/docker-tagging-best-practices-for-tagging-and-versioning-docker-images/

in the meanwhile, i think the `7` here is a typo?
```suggestion
- Run `echo "FROM samveralabs/hyrax:<tag>" > Dockerfile` to create the Dockerfile
```